packages/client-telegram/package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@ai16z/eliza": "workspace:*",
     "@telegraf/types": "7.1.0",
+    "node-fetch": "^3.3.2",
```

Comment:
Is the telegram client not using node? Otherwise this shouldn't be needed anymore

Reply:
not sure what you mean?

Reply:
fetch is included in node now (since v18). there's no need to install the fetch package itself